crates/store/re_log_encoding/src/rrd/encoder.rs

```diff
@@ -106,17 +99,20 @@ impl Encoder<Vec<u8>> {
 }
 
 impl<W: std::io::Write> Encoder<W> {
-    pub fn new(
+    pub fn new_eager(
```

Comment:
worth adding a docstring to hint at why you would want the `eager` version and what you'd do if you didn't. Or what this is not called `new` to being with.